cmd/finch/main_native.go

```diff
@@ -79,6 +79,9 @@ var newApp = func(
 	// TODO: Decide when to forward --debug to the dependencies
 	// (e.g. nerdctl for container commands and limactl for VM commands).
 	rootCmd.PersistentFlags().Bool("debug", false, "running under debug mode")
+	rootCmd.PersistentFlags().String("namespace", "", `containerd namespace, such as "moby" for Docker, "k8s.io" for Kubernetes`)
+	rootCmd.PersistentFlags().String("cgroup-manager", "", `Cgroup manager to use ("cgroupfs"|"systemd")`)
+	rootCmd.PersistentFlags().String("snapshotter", "", "containerd snapshotter")
```

Comment:
Does this have precedent over the finch.yaml snapshotter configuration?

Reply:
I think yes, according to the docs [here](https://github.com/runfinch/finch/blob/main/docs/snapshotters.md#snapshotters-option)
> The default snapshotter will be used whenever a snapshotter is needed and not specified for a command. Any other listed snapshotters can be used by specifying them with the --snapshotter flag